crates/rspack_core/src/tree_shaking/visitor.rs

```diff
@@ -536,24 +578,24 @@ impl<'a> Visit for ModuleRefAnalyze<'a> {
                 _ => object,
               },
               Part::Url(src) => {
-                let (src_module_id, dep_id) = self
+                let dep_id = self
                   .resolve_module_identifier(src, &DependencyType::NewUrl)
                   .unwrap_or_else(|| panic!("Can't resolve {} in {}", src, self.module_identifier));
 
                 return HashSet::from_iter([SymbolRef::Url {
                   importer: self.module_identifier,
```

Comment:
Here is why we can't use `Dependency` id directly, because we can't get the dependency id of the module id, one module id may correspond to many dependency id.